pkg/storage/bloom/v1/test_util.go

```diff
@@ -26,8 +26,10 @@ func MakeBlockQuerier(t testing.TB, fromFp, throughFp model.Fingerprint, fromTs,
 	builder, err := NewBlockBuilder(
 		BlockOptions{
 			schema: Schema{
-				version:  DefaultSchemaVersion,
-				encoding: chunkenc.EncSnappy,
+				version:     DefaultSchemaVersion,
+				encoding:    chunkenc.EncSnappy,
+				nGramLength: 4, // see DefaultNGramLength in bloom_tokenizer_test.go
+				nGramSkip:   0, // see DefaultNGramSkip in bloom_tokenizer_test.go
```

Comment:
why not directly use those defaults? or move them here?

Reply:
The test package cannot be imported into a regular package.

Reply:
ah sorry, got it.